packages/vscode-extension/package.json

```diff
@@ -490,6 +490,11 @@
         "title": "Teams - Validate Local Prerequisites",
         "enablement": "never"
       },
+      {
+        "command": "fx-extension.append-func-path",
```

Comment:
I think `get-func-path` is better since this command is just to get the path of func managed/detected by our toolkit. The appending operation is done in tasks.json but not in this command.

Reply:
Good idea. Updated to `get-func-path`